### PR TITLE
Fix to handle regression test when package is missing on PyPI

### PR DESCRIPTION
**Pull request by praveenkuttappan**
API to get versions available on PyPI for a package is throwing exception when package is missing on PyPI. This PR is to handle the scenario when version info is not available on PyPI.

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: scripts/devops_tasks/git_helper.py [hunk 1]
```diff
@@ -30,15 +30,20 @@ def get_release_tag(dep_pkg_name, isLatest):
...
     client = PyPIClient()
-    versions = [str(v) for v in client.get_ordered_versions(dep_pkg_name)]
-    logging.info("Versions for {0} is: {1}".format(dep_pkg_name, versions))
+    versions = []
+    try:
+        versions = [str(v) for v in client.get_ordered_versions(dep_pkg_name)]
+        logging.info("Versions available on PyPI for {0} are: {1}".format(dep_pkg_name, versions))
+    except:
+        logging.error("Package {} is not available on PyPI".format(dep_pkg_name))
+        return
```
Comment (scbedd) [inline scripts/devops_tasks/git_helper.py]: `None` for explicitness.

Comment (praveenkuttappan) [reply scripts/devops_tasks/git_helper.py]: sure. I will make this change.

File: scripts/devops_tasks/git_helper.py [hunk 2]
```diff
@@ -30,15 +30,20 @@ def get_release_tag(dep_pkg_name, isLatest):
...
     # filter excluded versions
     if dep_pkg_name in EXCLUDED_PACKAGE_VERSIONS:
         versions = [v for v in versions if  v not in EXCLUDED_PACKAGE_VERSIONS[dep_pkg_name]]
         logging.info("Filtered versions for {0} is: {1}".format(dep_pkg_name, versions))
 
-    if len(versions) == 0:
+    if not versions:
         logging.info(
             "Released version info for package {} is not available".format(dep_pkg_name)
         )
```
Comment (scbedd) [inline scripts/devops_tasks/git_helper.py]: Oh man. We even have a comment explicitly noting why it's ok to exit quietly. 🤦‍♂

Comment (praveenkuttappan) [reply scripts/devops_tasks/git_helper.py]: Yup. Didn't realize that versions API will throw HTTP exceptions instead of empty result set.

**Pull request merged**